Snippets/Snippets_5/Outbox/RavenDB/TimeToKeep.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using NServiceBus;
+using NServiceBus.Configuration.AdvanceExtensibility;
+
+public class TimeToKeep
+{
+    public void Customize()
+    {
+        BusConfiguration configuration = null;
+
+        #region OutboxRavendBTimeToKeep
+        configuration.GetSettings().Set("Outbox.TimeToKeepDeduplicationData", TimeSpan.FromDays(7));
```

Comment:
@SimonCropp we decided not to do it ecause of this discussion: https://github.com/Particular/NServiceBus/pull/2712#issuecomment-93671459